.github/workflows/macos.yml

```diff
@@ -122,24 +123,29 @@ jobs:
           cmake --version
 
       - name: Config and build
+        shell: bash -l {0}
         run: |
           PATH=/usr/local/var/homebrew/linked/ccache/libexec:$PATH
           ccache -s
           source util/ci_utils.sh
-          build_wheel
+          conda install conda-build=3.20.0 -y
```

Comment:
Can we move this inside `ci_utils.sh` `install_python_dependencies`, with version in `CONDA_BUILD_VER` variable at the top.

Reply:
I tried doing that in 4544dd7, however, this creates additional complexities for the GPU docker test, since the GPU test also runs `install_python_dependencies` while it is not running in a conda environment. Any suggestions?

Reply:
We can make it optional. `install_python_dependencies` already has optional like `with-cuda`. We can add `with-conda`.